client/v2/autocli/testdata/file.test

```diff
@@ -0,0 +1 @@
+this is just a test file
```

Comment:
Why is it just text? I would have expected an hex encoded string in a file.
Unless I am misunderstanding the issue (cc @aaronc)

Reply:
I understood as just if it is a file, you just save the bytes from this comment https://github.com/cosmos/cosmos-sdk/pull/15504#discussion_r1144945667.